components/article-list-item.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Link from 'next/link';
+import { string } from 'prop-types';
+
+const Item = (props) =>{
+  return (
+    <div className="card">
+      <Link href="/a/[slug]" as={`/a/${props.slug}`}>
```

Comment:
This adds the 'Link' functionality to the `columns` `div`. But if I'll add the `<a>` tag as a direct child, the browser will return an error saying: "a" cannot be a descendent of "a".

Reply:
It's happening because Next's  `<Link />` is already an anchor tag and the browser is not able to read nested <a> elements. IIRC, it will turn into two separate <a>'s.